### ISSUE TITLE
Long-term photometry

### ISSUE THREAD

**Issue by yaelnaze** [lukeshingles/atlasserver#55]
I entered a new request on https://fallingstar-data.com/forcedphot/queue/  using my star's coordinates. I put as minimum MJD a very low value (10 000) to be sure to get all data since the start of ATLAS. (I usually do something similar for ASAS-SN and it works) However, once the run was finished, I only got for about a hundred days in 2020 (59101-59185). How is that possible ? Shouldn't I get back the whole set of data used by Heinze et al. 2018, but complemented by new measurements since then ? Also, is there a saturation limit or a quality flag somewhere ? The m column seems quite erratic, with negative or very low values (m~20) while my star is quite bright (V~12).

**Comment by smarttgit** [comment]
Are you requesting forced photometry on the difference images (difference image = target image - reference image) ? In which case the measured flux `uJy` may well be close to zero if the star is not particularly variable, or it is a phase when the flux is similar to the flux in the reference image. You should expect the `uJy` flux to be small on many epochs. A small uJy may mean that the significance of the detection is low (less than 3 sigma), in which case the conversion to magnitudes is unreliable. 

There is no saturation flag in the returned forced photometry - the saturation limit will be around 11-12, so you should be careful. 

You can run the forced photometry on the target images (not the difference images), but beware of the results - check our FAQ. In all case just think of what measurement we are providing, it is not always the same as you are expecting. 

If the star is intrinsically variable, it may be in our First data release : [(https://ui.adsabs.harvard.edu/abs/2018AJ....156..241H/abstract)]

**Comment by smarttgit** [comment]
The main measure of quality is the value of : 
**chi/N	reduced chi^2 of the PSF fit**
https://fallingstar-data.com/forcedphot/resultdesc/

**Comment by yaelnaze** [comment]
Yes, it's in the Heinze paper - I had problems accessing the database, and I was also hoping to find more recent data, explaining why I tried the forced photometry. Yes, I'm close to the saturation (so it's well over a 3sigma detection). besides, variations are not so small (0.2 mag). I was quite surprised because in things like ASAS-SN, forced photometry works quite well...

**Comment by smarttgit** [comment]
Also linking the photometry of a source the Heinze paper with the results from the forced server, over the same period of time, would be useful to see

**Comment by yaelnaze** [comment]
fully agree it'd be useful, i'll try and send results if meaningful.
